src/platform/packages/shared/kbn-scout/test/scout/tests/api_services/fleet.spec.ts

```diff
@@ -0,0 +1,493 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { apiTest, expect } from '../../../../src/playwright';
+
+apiTest.describe('Fleet API Service', { tag: ['@svlSecurity', '@ess'] }, () => {
+  apiTest.describe('Integration Management', () => {
+    let integrationName: string;
+
+    apiTest.beforeEach(async () => {
+      integrationName = `test-integration-${Date.now()}`;
+    });
+
+    apiTest.afterEach(async ({ apiServices }) => {
+      // Clean up integration
+      try {
+        await apiServices.fleet.integration.delete(integrationName);
+      } catch (e) {
+        // Integration might not exist or already deleted
+      }
+    });
+
+    apiTest('should install a custom integration', async ({ apiServices }) => {
+      const response = await apiServices.fleet.integration.install(integrationName);
+
+      expect(response.status).toBe(200);
+      expect(response.data).toBeDefined();
+    });
+
+    apiTest('should delete an integration and return status code', async ({ apiServices }) => {
+      // First install the integration
+      await apiServices.fleet.integration.install(integrationName);
+
+      // Then delete it
+      const statusCode = await apiServices.fleet.integration.delete(integrationName);
```

Comment:
for consistency and future extension: let's return object {status: number} for both as a minimum